docs/monitoring-usage.rst

```diff
@@ -278,3 +278,112 @@ follows::
+
+When creating :class:`~gcloud.monitoring.timeseries.TimeSeries`, you specify points containing a
+value of the type that matches the *value_type* specified in the associated `MetricDescriptor`.
+The point must also contain a time interval. The point's time interval must be later than any
+points already created for the same :class:`~gcloud.monitoring.timeseries.TimeSeries` (where
+:class:`~gcloud.monitoring.timeseries.TimeSeries` with matching :class:`~gcloud.monitoring.metric
+.Metric` and :class:`~gcloud.monitoring.metric.Resource` types are considered the same). The end
+time of the interval must not be more than 24 hours in the past or more than five minutes in the
+future::
+
+    >>> start = datetime.utcnow()
+    >>> end = datetime.utcnow() - datetime.timedelta(minutes=5)
+    >>> value = get_num_404s_in_last_five_minutes() # get value from a hypothetical call
+    >>>
+    >>> client.write_timeseries(value, start_time=start, end_time=end,
+    ...                         metric=metric, resource=resource) # API call
+
+You can also use the :class:`~gcloud.monitoring.TimeseriesWriter` class to
+associate a fully specified Metric and Resource pair to write :class:`~gcloud.monitoring.timeseries.TimeSeries` together::
+
+    >>> timeseries_writer = client.get_timeseries_writer(metric, resource)
+    >>> timeseries_writer.write_point(value, start_time=start, end_time=end) # API call
+
+Stackdriver Monitoring supports several **value types**. A *CUMULATIVE* metric represents a value
+over time. Successive points in a :class:`~gcloud.monitoring.timeseries.TimeSeries` should have the
+same start time and increasing end times, until an event resets the cumulative value to zero and
+sets a new start time for the follow points.
+
+For *GAUGE* metrics, the time intervals a single point in time, so only the end_time should be
+specified::
+
+    >>> timeseries_writer.write_point(value, end_time=end) # API call
+
+By default, end_time defaults to :meth:`~datetime.datetime.utcnow()`, so metrics written for the
+can be simplified to::
+
+   >>> timeseries_writer.write_point(value) # API call
+
+For *DELTA* metrics, the points reflect a change during a finite time interval, so both start_time
+and end_time should be specified, though end_time again defaults to the current time.
+
+    >>> timeseries_writer.write_point(value, start_time=start_time) # API call
+
+It's also possible to have the :class:`gcloud.monitoring.client.Client` return a
+:class:`gcloud.monitoring.timeseries.TimeSeries` object, which can then written later::
+
+    >>> ts1 = client.timeseries(value, end_time=end_time, metric=metric1, resource=resource1)
+    >>> ts1.create()
+
+or be used in :meth:`~gcloud.monitoring.client.write_time_series`::
+
+    >>> ts1 = client.timeseries(value, end_time=end_time, metric=metric1, resource=resource1)
+    >>> ts2 = client.timeseries(value2, end_time=end_time2, metric=metric2, resource=resource2)
+    >>> client.write_time_series([ts1, ts2])
```

Comment:
I was going to say  the name is very similar to `write_timeseries` above, but see that the above method was to be `write_point`.

In any case, it is better to drop this method from the client, and perhaps add it to `TimeSeriesWriter`? We could even not support this for now, since the common use case is to set up a single metric stream, and write points to it.